doc/classes/Control.xml

```diff
@@ -409,6 +409,7 @@
 			</return>
 			<description>
 				Steal the focus from another control and become the focused control (see [member focus_mode]).
+				[b]Note:[/b] This method may not always work when called directly (it may do nothing). As a workaround, defer the call by one frame using [code]call_deferred("grab_focus")[/code] instead. This won't block your script's execution.
```

Comment:
It seems weird to write "This method may not always work when called directly (it may do nothing).".
Is it non-deterministic, or just moody? We should be able to describe in what case it behaves like this, so that users know when they need to delay the call instead of relying on try and error.

Reply:
Also it may be worthwhile to see if we could make this the default then automatically.

Reply:
It's likely not that it does not work, but more that if it is executed before other focus grabbing actions (like handling mouse motion events), it might be "cancelled" during the same frame.
I'm not against mentioning the workaround, but the first part of the sentence should be rephrased.

Reply:
Any suggestions on how to rephrase?

I guess we need to go deeper to understand when it's not effective to be able to write correct documentation here.

Reply:
We could also make the method deferred by default. Is there any need for immediate focus?

Reply:
Heh, so I just accidentally created a duplicate PR that phrases it a bit differently: #59855 idk if it's good enough.